libs/sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -582,6 +582,30 @@ public void testSetLicense() {
         assertThat(assignment.expressions().get(0).toString()).isEqualTo("'LICENSE_KEY'");
     }
 
+    @Test
+    public void test_set_timezone_value() {
+        SetStatement<?> stmt = (SetStatement<?>) SqlParser.createStatement("SET TIME ZONE 'Europe/Vienna'");
+        assertThat(stmt.scope()).isEqualTo(SetStatement.Scope.TIME_ZONE);
+        assertThat(stmt.assignments()).hasSize(1);
+        assertThat(stmt.assignments().get(0).expressions().get(0).toString()).isEqualTo("'Europe/Vienna'");
+    }
+
+    @Test
+    public void test_set_timezone_local() {
+        SetStatement<?> stmt = (SetStatement<?>) SqlParser.createStatement("SET TIME ZONE LOCAL");
```

Comment:
maybe you could consolidate them in one method, wdyt?

Reply:
Yes sure. I got the feedback in the past to split tests up, but no strong opinion on this.

Reply:
Separate test cases make it easier to troubleshoot issues. No reason to fold them together if the test-setup is cheap (which is the case here)

Reply:
It's fine to keep it as is, also no strong opinion, just a slight preference since all tests test happy path, just with different values.